capstone/src/main/webapp/cover.css

```diff
@@ -0,0 +1,107 @@
+/*
+ * Globals
+ */
+
+/* Links */
+a,
+a:focus,
+a:hover {
+  color: #fff;
+}
+
+/* Custom default button */
+.btn-secondary,
+.btn-secondary:hover,
+.btn-secondary:focus {
+  color: #fff;
+  text-shadow: none; /* Prevent inheritance from `body` */
+  background-color: #451735;
+  border: .05rem solid #fff;
+}
+
+/*
+ * Base structure
+ */
+
```

Comment:
Nit: The space here should be removed for consistency in comments

Reply:
addressed in "address nits"